.travis.yml

```diff
@@ -71,6 +72,9 @@ before_install:
   # Remove Ruby directive from Gemfile as this image has 2.2.5
   - sed -i '/^ruby/d' Gemfile
   - mkdir -p ~/.apt-cache
+  - sudo add-apt-repository -y ppa:marutter/rdev
+  - sudo add-apt-repository -y ppa:staticfloat/juliareleases
+  - sudo add-apt-repository -y ppa:ondrej/golang
```

Comment:
well this was just me trying to review the pr despite having very little experience with what you did :) 
Soooo I can't really answer your question. Maybe put it there but if having them in deps.sh is expected/wanted then I see no need to point it out.